### ISSUE TITLE
bug with automatic drop collection

### ISSUE THREAD

**Issue by xperisa** [Zaarrg/TTVDropBot#77]
GQL RESPONSE ERROR! DropsPage_ClaimDropRewards Request Failed... Retrying in 30 seconds... Try: 1/4
GQL RESPONSE ERROR! DropsPage_ClaimDropRewards Request Failed... Retrying in 30 seconds... Try: 2/4
GQL RESPONSE ERROR! DropsPage_ClaimDropRewards Request Failed... Retrying in 30 seconds... Try: 3/4
GQL RESPONSE ERROR! DropsPage_ClaimDropRewards Request Failed... Retrying in 30 seconds... Try: 4/4

**Comment by Voltzy** [comment]
Read through the other issues before posting one.
This is an issue every drop bot has been experiencing for about a couple of weeks since twitch added a new anti-bot thing.

**Comment by xperisa** [comment]
> 在发布之前通读其他问题。 自从 twitch 添加了一个新的反机器人功能以来，这是每个 drop bot 都经历了大约几周的问题。

 any bots that solve this problem?

**Comment by Voltzy** [comment]
> > 在发布之前通读其他问题。 自从 twitch 添加了一个新的反机器人功能以来，这是每个 drop bot 都经历了大约几周的问题。
> 
> any bots that solve this problem?

No. They are all broken.

**Comment by cyberofficial** [comment]
# Patch Release 
* Build [00.01](https://github.com/cyberofficial/TTVDropBot/releases/tag/00.01) by @cyberofficial
     * Commit https://github.com/cyberofficial/TTVDropBot/commit/f7f3cc42fa97d7ecba93015f99dce1c56c2d7754
          * Removed channel point error and skips over it.


You can also always download the forked repo and run in linux aswell.

Temporal patch until a real fix is delivered., Hope this helps people in long run. 
If you have issues with the patch, make report on the fork repo so the main owner isn't disturbed for my patches.
I do not mind working on this project since I'm using it too.

It does work for me on windows x64 (windows 11).
![](https://i.imgur.com/0PtJIpL.png)

**Comment by hectorevil** [comment]
> # Patches
> * [Releases](https://github.com/cyberofficial/TTVDropBot/releases)
> 
> You can also always download the forked repo and run in linux aswell.
> 
> Temporal patch until a real fix is delivered., Hope this helps people in long run. If you have issues with the patch, make report on the fork repo so the main owner isn't disturbed for my patches. I do not mind working on this project since I'm using it too.
> 
> It does work for me on windows x64 (windows 11). ![](https://camo.githubusercontent.com/17feae3ed422897bcfcfd7f2ae04e817579223f3b8edd13fda564fdc1b1ccd78/68747470733a2f2f692e696d6775722e636f6d2f3050744a49704c2e706e67)

Tried your Drop patched but it keeps closing for no apparent reason

**Comment by Voltzy** [comment]
> > # Patches
> > 
> > * [Releases](https://github.com/cyberofficial/TTVDropBot/releases)
> > 
> > You can also always download the forked repo and run in linux aswell.
> > Temporal patch until a real fix is delivered., Hope this helps people in long run. If you have issues with the patch, make report on the fork repo so the main owner isn't disturbed for my patches. I do not mind working on this project since I'm using it too.
> > It does work for me on windows x64 (windows 11). ![](https://camo.githubusercontent.com/17feae3ed422897bcfcfd7f2ae04e817579223f3b8edd13fda564fdc1b1ccd78/68747470733a2f2f692e696d6775722e636f6d2f3050744a49704c2e706e67)
> 
> Tried your Drop patched but it keeps closing for no apparent reason

yeah twitch login is being funky with these bots and causing it to close.

**Comment by cyberofficial** [comment]
If fixed some more issues, you can try [00.03](https://github.com/cyberofficial/TTVDropBot/releases/tag/00.02-hotfix) from me see if that helps, make sure to do the new twitch session method aswell, and disable channel points collection also shown how to do that

**Comment by TcNobo** [comment]
I am also getting this error... Sad because there's Rust drops I'd like to get

**Comment by cyberofficial** [comment]
> I am also getting this error... Sad because there's Rust drops I'd like to get

Could be an issue with your auth-token, get a new auth token using my method, worked for me and few friends that also the issue

**Comment by TcNobo** [comment]
Yep! A fresh token fixed it. All I needed to do.

**Comment by iAnvar** [comment]
> > I am also getting this error... Sad because there's Rust drops I'd like to get
> 
> Could be an issue with your auth-token, get a new auth token using my method, worked for me and few friends that also the issue

Tried your Drop patched but still gets error, can you help?
![image](https://user-images.githubusercontent.com/84909834/195260586-7c212b79-2aeb-4d2b-93f5-b769f6a73dfb.png)

**Comment by cyberofficial** [comment]
> > > I am also getting this error... Sad because there's Rust drops I'd like to get
> > 
> > 
> > Could be an issue with your auth-token, get a new auth token using my method, worked for me and few friends that also the issue
> 
> Tried your Drop patched but still gets error, can you help? ![image](https://user-images.githubusercontent.com/84909834/195260586-7c212b79-2aeb-4d2b-93f5-b769f6a73dfb.png)

I'm really confused on how it works for some and broken for others, wtf did twitch change in their back end, like fook.
All these issues and I only encountered the twitch points error, all these new issues are weird.

I'm looking into fixing [gql-ttvdropbot](https://github.com/Zaarrg/twitch-gql-ttvdropbot) and seeing what I can do with that.
As of right now, It's a stand still of wtf twitch why are they breaking everything 🥲 and BTTV and FFZ are suffering too.

**Comment by BlackThenWhite** [comment]
> > > > I am also getting this error... Sad because there's Rust drops I'd like to get
> > > 
> > > 
> > > Could be an issue with your auth-token, get a new auth token using my method, worked for me and few friends that also the issue
> > 
> > 
> > Tried your Drop patched but still gets error, can you help? ![image](https://user-images.githubusercontent.com/84909834/195260586-7c212b79-2aeb-4d2b-93f5-b769f6a73dfb.png)
> 
> I'm really confused on how it works for some and broken for others, wtf did twitch change in their back end, like fook. All these issues and I only encountered the twitch points error, all these new issues are weird.
> 
> I'm looking into fixing [gql-ttvdropbot](https://github.com/Zaarrg/twitch-gql-ttvdropbot) and seeing what I can do with that. As of right now, It's a stand still of wtf twitch why are they breaking everything 🥲 and BTTV and FFZ are suffering too.
> 
> I'm using the program, and I want to try and contribute fixes; but its twitch going 1 step forward, 2 steps back with their code and im trying to go 5 steps ahead

I'm using your forked version along with the instructions posted on your github page, along with (https://chrome.google.com/webstore/detail/automatic-twitch-drops-mo/kfhgpagdjjoieckminnmigmpeclkdmjm/)
 and keeping the (https://www.twitch.tv/drops/inventory)  tab open and it's working perfectly fine. The chrome addon seems to have been fixed to claim drops and your patched version along with the settings edit is bypassing the error and watching streams fine.
